File: Modul 5/Python/PRAK504-2310817310010-AkmallullailSyaban.py
def reverse(nilai):
    reversed = 0
    while nilai != 0:
        digit = nilai % 10
        reversed = reversed * 10 + digit
        nilai //= 10
    return reversed

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))

File: Modul 5/C/PRAK504-2310817310010-AkmallullailSyaban.c
#include <stdio.h>
int reverse(int nilai){
// Lengkapi Function ini
    int reversed = 0;
    while (nilai != 0){
        int digit = nilai % 10;
        reversed = reversed * 10 + digit;
        nilai /= 10;
    }
    return reversed;
}

int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));
}